### PR TITLE
Add presentedForm.title to DiagnosticReport

### PR DESCRIPTION
**Pull request by adamb3ard**
DiagnosticReport now returns a title in PresentedForm, so here is an updated Overview and JSON example.

Pics of changes running locally:
Overview:
DiagRepOverview

JSON example:
DiagRepJSON

### CODE REVIEW

**Comment by adamb3ard** [comment]
Pic of overview with PresentedForm fields nested under it:

nestDiagRep

File: content/millennium/dstu2/diagnostic/diagnostic-report.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ The following fields are returned if valued:
 * [Issued date/time](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.issued){:target="_blank"}
 * [Performer (practitioner)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.performer){:target="_blank"}
 * [Request (order)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.request){:target="_blank"}
-* [ContentType and URL (fully qualified link to the document](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.presentedForm){:target="_blank"}
+* PresentedForms:
+  * [ContentType, URL, and Title (fully qualified link to the document)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.presentedForm){:target="_blank"}
```
Comment (averyallen) [inline content/millennium/dstu2/diagnostic/diagnostic-report.md]: In R4, we've been breaking up subfields individually. [Example for DocRef](http://fhir.cerner.com/millennium/r4/documents/document-reference/). Do we want to do that for dstu2 as well?

@MaxPhilips

Comment (MaxPhilips) [reply content/millennium/dstu2/diagnostic/diagnostic-report.md]: i PMed adam asking for this :)

**Comment by adamb3ard** [comment]
@MaxPhilips  @averyallen 
[Updated here](https://github.com/cerner/fhir.cerner.com/pull/338/commits/0992908621c8ebd4207546d8463af9253f5968c1)

Pic with changes:

Screen Shot 2020-01-10 at 12 28 22 PM

File: content/millennium/dstu2/diagnostic/diagnostic-report.md [hunk 2]
```diff
@@ -26,7 +26,10 @@ The following fields are returned if valued:
 * [Issued date/time](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.issued){:target="_blank"}
 * [Performer (practitioner)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.performer){:target="_blank"}
 * [Request (order)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.request){:target="_blank"}
-* [ContentType and URL (fully qualified link to the document](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.presentedForm){:target="_blank"}
+* [Presented Form](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.presentedForm){:target="_blank"}
+  * [ContentType](http://hl7.org/fhir/DSTU2/datatypes-definitions.html#Attachment.contentType){:target="_blank"}
```
Comment (MaxPhilips) [inline content/millennium/dstu2/diagnostic/diagnostic-report.md]: the other one says Attachment ContentType, probably we should just change both to 'Content type'

Comment (adamb3ard) [reply content/millennium/dstu2/diagnostic/diagnostic-report.md]: [Updated](https://github.com/cerner/fhir.cerner.com/pull/338/commits/a74a052b80d6ebd14b13ee8fb007ba0a561426c2)

File: content/millennium/dstu2/diagnostic/diagnostic-report.md (hunk 1 above)
Comment (adamb3ard) [reply content/millennium/dstu2/diagnostic/diagnostic-report.md]: [Update in this commit](https://github.com/cerner/fhir.cerner.com/pull/338/commits/0992908621c8ebd4207546d8463af9253f5968c1#diff-663275837ef95b5d96d65014e7c0e943R29-R32)

**Pull request merged**